front/migrations/db/migration_273.sql

```diff
@@ -0,0 +1,31 @@
+-- Create temp table
+CREATE TEMP TABLE workos_mapping (
+    workos_user_id VARCHAR(255),
+    auth0_user_id VARCHAR(255),
+    created BOOLEAN
+);
+
+-- Load CSV from disk
+\copy workos_mapping FROM 'migrations/db/migration-result-auth0.csv' WITH CSV HEADER;
+
+-- Verify data loaded correctly
+SELECT COUNT(*) FROM workos_mapping;
+
+-- Run the backfill update
+UPDATE users
+SET "workOSUserId" = wm.workos_user_id
+FROM workos_mapping wm
+WHERE users."workOSUserId" IS NULL
+  AND users."auth0Sub" = wm."auth0_user_id"
+  AND users.email IN (
+    SELECT email
+    FROM users
+    GROUP BY email
+    HAVING COUNT(*) = 1
+  );
+
+-- Check how many rows were updated
+SELECT COUNT(*) FROM users WHERE "workOSUserId" IS NOT NULL;
```

Comment:
this counts the number of total workOSUserId right, this is not the count of how many were updated ?

if we wanted we could maybe verify that by doing a select before the backfill to count how many are planned to be updated / how many already exist

idk if pertinent

Reply:
Can definitely count the number of rows that are null before the update